geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AvailableTables.scala

```diff
@@ -9,11 +9,12 @@
 package org.locationtech.geomesa.accumulo.data.tables
 
 object AvailableTables {
```

Comment:
ah, maybe... i don't think it will matter as the ST table will not `supports` any newly created schema. Would changing it affect old schemas that don't have explicitly configured tables though?